packages/atomic-angular/projects/atomic-angular/src/lib/stencil-generated/components.ts

```diff
@@ -161,6 +161,28 @@ export class AtomicCommerceInterface {
 export declare interface AtomicCommerceInterface extends Components.AtomicCommerceInterface {}
 
 
+@ProxyCmp({
```

Comment:
Mark the atomic component `@internal` in JSDoc, and do a rebuild: This should clean up those generated files.

For the time being I would prefer we don't output angular/react until we're more mature with Atomic commerce.